atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -323,67 +327,14 @@ public static TransactionManager create(
                         new NamedThreadFactory("AsyncInitializer-SerializableTransactionManager", true)),
                 validateLocksOnReads,
                 transactionConfig,
-                conflictTracer,
-                metricsFilterEvaluationContext,
-                sharedGetRangesPoolSize);
-    }
-
-    public static TransactionManager create(
```

Comment:
Large internal product uses constructor above - no reason to continue to maintain this one.